pages/recurring-contributions.js

```diff
@@ -153,6 +155,13 @@ class recurringContributionsPage extends React.Component {
                     values={I18nFormatters}
                   />
                 )}
+                {notificationType === 'update' && (
+                  <FormattedMessage
+                    id="subscription.createSuccessUpdated"
+                    defaultMessage="<strong>Your recurring contribution has been updated.</strong>"
+                    values={I18nFormatters}
+                  />
```

Comment:
If the entire value is wrapped in `<strong>` (and thus the tag has no incidence on the translation) it's better to not include it in the message:

```suggestion
                  <strong>
                    <FormattedMessage id="subscription.createSuccessUpdated" defaultMessage="Your recurring contribution has been updated." />
                  </strong>
```